frontend/fingertips-frontend/components/organisms/LineChart/lineChart.test.tsx

```diff
@@ -1,21 +1,54 @@
 import { render, screen } from '@testing-library/react';
 import { LineChart } from '@/components/organisms/LineChart/index';
 import { expect } from '@jest/globals';
-import { WeatherForecast } from '@/generated-sources/api-client';
 import { registryWrapper } from '@/lib/testutils';
+import { HealthCareData } from '@/app/chart/health-data';
 
-const mockData: WeatherForecast[] = [
+const mockData: HealthCareData[] = [
   {
-    date: new Date('2024-11-01T00:00:00.000Z'),
-    temperatureC: -30,
-    temperatureF: -21,
-    summary: 'Freezing',
+    areaCode: 'A1426',
+    healthData: [
+      {
+        year: 2023,
+        count: 222,
+        value: 506.60912,
+        lowerCi: 441.69151,
+        upperCi: 578.32766,
+      },
+      {
+        year: 2023,
+        count: 222,
+        value: 506.60912,
+        lowerCi: 441.69151,
+        upperCi: 578.32766,
+      },
+      {
+        year: 2023,
+        count: 222,
+        value: 506.60912,
+        lowerCi: 441.69151,
+        upperCi: 578.32766,
+      },
+    ],
   },
   {
-    date: new Date('2024-11-01T00:00:00.000Z'),
-    temperatureC: 0,
-    temperatureF: 32,
-    summary: 'Bracing',
+    areaCode: 'A1426',
```

Comment:
I know this is just mock data but is this accurate in terms of the areaCode being the same as the first element. Perhaps it would be better to use a different area code here.